### PR TITLE
DOC-473: Added dependencies to reissuance docs.

### PR DESCRIPTION
**Pull request by taylorharvey-r3**

### CODE REVIEW

File: content/en/platform/corda/4.8/open-source/reissuing-states.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ The new state reissuance functionality provides a state reissuance algorithm tha
 {{< note >}}
 State encumbrance refers to a state pointing to another state that must also appear as an input to any transaction consuming this state. A state may be encumbered by up to one other state, which is called an "encumbrance" state. The encumbrance state, if present, forces additional controls over the encumbered state, since the encumbrance state contract will also be verified during the execution of the transaction.
 
-See [Defining encumbrances](../../../../tutorials/corda/4.8/os/build-basic-cordapp/basic-cordapp-contract.md) for more information.
+See [Encumbrances](../../../../en/platform/corda/4.8/open-source/key-concepts-contracts.html#encumbrances) for more information.
```
Comment (taylorharvey-r3) [inline content/en/platform/corda/4.8/open-source/reissuing-states.md:37]: NB this link fix is something I worked on recently. It's not in the scope of this ticket, but I saw it while I was here (and checked in other versions, this is the only version that needed to be fixed).

File: content/en/platform/corda/4.7/enterprise/cordapps/reissuing-states.md [hunk 2]
```diff
@@ -47,6 +47,13 @@ This functionality is part of Corda open source 4.7 and can be fully leveraged b
 
 A trusted issuer reissues an encumbered state before the original state is deleted, thus enabling the requester to unlock that reissued state immediately after the original state is deleted.
 
+### Dependencies
+
+The Reissuance CorDapp has the following dependencies:
+
+* Corda Open Source or Corda Enterprise, version 4.7 or higher
+* [Tokens SDK](https://github.com/corda/token-sdk)
```
Comment (erinking-r3) [inline content/en/platform/corda/4.7/enterprise/cordapps/reissuing-states.md:55]:
```suggestion
### Reissuance CorDapp dependencies

* Corda Open Source or Corda Enterprise, version 4.7 or higher
* [Tokens SDK](https://github.com/corda/token-sdk)
```

File: content/en/platform/corda/4.7/open-source/reissuing-states.md [hunk 3]
```diff
@@ -47,6 +47,13 @@ This functionality is part of Corda open source 4.7 and can be fully leveraged b
 
 A trusted issuer reissues an encumbered state before the original state is deleted, thus enabling the requester to unlock that reissued state immediately after the original state is deleted.
 
+### Dependencies
+
+The Reissuance CorDapp has the following dependencies:
```
Comment (erinking-r3) [inline content/en/platform/corda/4.7/open-source/reissuing-states.md:52]:
```suggestion
### Reissuance CorDapp dependencies
```

File: content/en/platform/corda/4.8/enterprise/cordapps/reissuing-states.md [hunk 4]
```diff
@@ -46,6 +46,13 @@ This functionality is part of Corda open source 4.8 and can be fully leveraged b
 
 A trusted issuer reissues an encumbered state before the original state is deleted, thus enabling the requester to unlock that reissued state immediately after the original state is deleted.
 
+### Dependencies
+
+The Reissuance CorDapp has the following dependencies:
+
```
Comment (erinking-r3) [inline content/en/platform/corda/4.8/enterprise/cordapps/reissuing-states.md:52]:
```suggestion
### Reissuance CorDapp dependencies

```

File: content/en/platform/corda/4.8/open-source/reissuing-states.md [hunk 5]
```diff
@@ -47,6 +47,13 @@ This functionality is part of Corda open source 4.8 and can be fully leveraged b
 
 A trusted issuer reissues an encumbered state before the original state is deleted, thus enabling the requester to unlock that reissued state immediately after the original state is deleted.
 
+### Dependencies
+
+The Reissuance CorDapp has the following dependencies:
```
Comment (erinking-r3) [inline content/en/platform/corda/4.8/open-source/reissuing-states.md:52]:
```suggestion
### Reissuance CorDapp dependencies
```

File: content/en/platform/corda/4.7/enterprise/cordapps/reissuing-states.md [hunk 6]
```diff
@@ -47,6 +47,13 @@ This functionality is part of Corda open source 4.7 and can be fully leveraged b
 
 A trusted issuer reissues an encumbered state before the original state is deleted, thus enabling the requester to unlock that reissued state immediately after the original state is deleted.
 
+### Dependencies
+
+The Reissuance CorDapp has the following dependencies:
```
Comment (erinking-r3) [inline content/en/platform/corda/4.7/enterprise/cordapps/reissuing-states.md:52]:
```suggestion
### Reissuance CorDapp dependencies
```

**Review by erinking-r3 — COMMENTED** [review]
Sorry to be a PITA but I'd suggest this change to create a more meaningful header and remove unnecessary text

**Review by erinking-r3 — APPROVED** [review]

**Pull request merged**